front/cypress/e2e/native_survey_page_action.cy.ts

```diff
@@ -116,26 +131,22 @@ describe('Native survey project page actions', () => {
     // Visit timeline project
     cy.visit(`/projects/${projectSlugTimeline}`);
     // Check that correct text and actions shown
+    cy.get('#e2e-cta-button').should('be.visible');
     cy.get('#e2e-cta-button').find('button').click({ force: true });
     cy.contains('New submissions can only be added in active phases.').should(
       'exist'
     );
   });
 
   it('tests actions when survey is not accepting submissions', () => {
-    // Login as admin
-    cy.setAdminLoginCookie();
-    // Visit admin project settings
-    cy.visit(`admin/projects/${projectIdContinuous}/native-survey`);
-    // Disable accepting submissions
-    cy.get('[type="checkbox"]').check();
```

Comment:
Seems like it could get into a state where the checkbox could be either value, so this check failed. I created a separate project instead just for this posting disabled case to make sure it can't happen.